newIDE/app/src/SceneEditor/index.js

```diff
@@ -1397,8 +1401,11 @@ export default class SceneEditor extends React.Component<Props, State> {
   };
 
   onSelectAllInstancesOfObjectInLayout = (objectName: string) => {
-    const { layout } = this.props;
-    const instancesToSelect = getInstancesInLayoutForObject(layout, objectName);
+    const { initialInstances } = this.props;
+    const instancesToSelect = getInstancesInLayoutForObject(
```

Comment:
I wonder if the method should have a different name now we don't give it a layout but an instancesContainer